packages/node_modules/nav-frontend-skjema/md/skjemagruppe/Skjemagruppe.overview.mdx

```diff
@@ -135,6 +135,26 @@ Skjemagrupper brukes vanligvis for å gruppere flere radioknapper eller checkbox
 </SkjemaGruppe>
 ```
 
+## Felles feilmelding uten propagering av feil til barnekomponenter
```

Comment:
Foreslår å endre denne til en `h3` som er underordnet den forrige overskriften "Felles feilmelding". Da kan vi også forenkle ordlyden en smule - noe ala:

```
### Uten propagering av feil til children
```

Reply:
![h3_endring](https://user-images.githubusercontent.com/26967723/78554556-a798be00-780b-11ea-8fa0-e26bf81d1605.JPG)

Noe slikt?

Reply:
Jau 👍 Men synes vi gjerne kan skrive "children" i stedet for "barnekomponenter". Ser egentlig ikke noe grunn til å oversette tekniske begreper som dette til norsk. Vi bruker f.eks. allerede engelsk på det tekniske begrepet "Legend" litt lengre opp på siden 🙂

Reply:
Enig. Fjerner all bruk av "barnekomponenter" over "children". 

![h3_endring_2](https://user-images.githubusercontent.com/26967723/78556609-905bcf80-780f-11ea-963a-3a564eab929e.jpg)